GTSAM was built against a different version of Eigen

**Description:**
I built gtsam with and without the default eigen. Still getting the error

**Console output:**
```
In file included from /home/harshal/vio_ws/devel/include/gtsam/base/Matrix.h:26:0,
                 from /home/harshal/vio_ws/devel/include/gtsam/base/Manifold.h:22,
                 from /home/harshal/vio_ws/devel/include/gtsam/base/Lie.h:25,
                 from /home/harshal/vio_ws/devel/include/gtsam/base/VectorSpace.h:11,
                 from /home/harshal/vio_ws/devel/include/gtsam/geometry/Point2.h:20,
                 from /home/harshal/vio_ws/devel/include/gtsam/geometry/Cal3_S2.h:24,
                 from /home/harshal/vio_ws/src/Kimera-VIO/include/kimera-vio/StereoFrame.h:35,
                 from /home/harshal/vio_ws/src/Kimera-VIO/include/kimera-vio/StereoVisionFrontEnd.h:25,
                 from /home/harshal/vio_ws/src/Kimera-VIO/src/StereoVisionFrontEnd.cpp:16:
/home/harshal/vio_ws/devel/include/gtsam/base/Vector.h:72:1: error: static assertion failed: Error: GTSAM was built against a different version of Eigen
 static_assert(
```
Similar error is thrown for other library files.



Please give also the following information:
* Operating system and version (e.g. Ubuntu 16.04 or Windows 10): ubuntu 16.04 with ROS kinetic
* Did you change the source code? (yes / no):  no

--- a/README.md
+++ b/README.md
@@ -57,6 +57,8 @@ mkdir -p ~/catkin_ws/src
 cd ~/catkin_ws/
 catkin init
 catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release
+# On Ubuntu 16.04:
+# catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release -DGTSAM_USE_SYSTEM_EIGEN=ON
 catkin config --merge-devel
 
 # Add workspace to bashrc for automatic sourcing of workspace.

